EIPS/eip-7870.md

```diff
@@ -0,0 +1,145 @@
+---
+eip: 7870
+title: Hardware and Bandwidth Recommendations for Validators and Full Nodes
```

Comment:
For the EIP title, I think we should remove "for validators and full nodes" for the following reasons:

1. The current name is too long. The max is 44 characters, this is 68. With that part it is 38 characters.
2. It's sort of implied that we're talking about validators & full nodes. But I suppose we could add recommendations for light clients too. A somewhat ambiguous title would allow us to do that.